frontend/controller/actions/identity.js

```diff
@@ -62,16 +62,6 @@ export default (sbp('sbp/selectors/register', {
     data: { username, email, password, picture },
     publishOptions
   }) {
-    // TODO: make sure we namespace these names:
-    //       https://github.com/okTurtles/group-income/issues/598
-    const oldSettings = await sbp('gi.db/settings/load', username)
-    if (oldSettings) {
-      // TODO: prompt to ask user before deleting and overwriting an existing user
-      //       https://github.com/okTurtles/group-income/issues/599
-      console.warn(`deleting settings for pre-existing identity ${username}!`, oldSettings)
-      await sbp('gi.db/settings/delete', username)
-    }
-
```

Comment:
Why was this removed?

Reply:
No longer needed as user settings are now indexed by contract ID, so there are no possible conflicts when creating a new account.